packages/tables/resources/views/index.blade.php

```diff
@@ -658,7 +658,7 @@ class="filament-tables-record-action-button flex-1 block py-3"
                                 :alignment="$column->getAlignment()"
                                 :sortable="$column->isSortable() && (! $isReordering)"
                                 :sort-direction="$getSortDirection()"
-                                :class="$getHiddenClasses($column)"
+                                class="filament-table-cell-{{ Str::of($column->getName())->camel()->kebab() }} {{ $getHiddenClasses($column) }}"
```

Comment:
Possibly move it into `@php` in the loop to avoid repetition?

```suggestion
                                class="filament-table-cell-{{ \Illuminate\Support\Str::of($column->getName())->camel()->kebab() }} {{ $getHiddenClasses($column) }}"
```

Reply:
It's spread across multiple foreach loops. We could extract it to a method though.

Reply:
Oh, potentially. Let's see what @danharrin thinks.

Reply:
Its fine how it is, but we cant use `str()` as that's Laravel 9 only. Must be `\Illuminate\Support\Str::of()`

Reply:
I already updated that part. Do you want the FQN instead of the alias everywhere?

Reply:
yes please, no aliases

Reply:
Done.